web/client/components/plugins/enhancers/withModulePlugins.js

```diff
@@ -42,7 +42,11 @@ const withModulePlugins = (getPluginsConfigCallback = getPluginsConfig) => (Comp
         if (!loading) onLoaded(true);
     }, [loading]);
 
-    return loading ? <Loader /> : <Component {...props} pluginsConfig={pluginsConfig} plugins={parsedPlugins} allPlugins={plugins} />;
+    if (loading && typeof loaderComponent === 'function') {
```

Comment:
...So if present, it is rendered (not always a function). 
```suggestion
    if (loading && loaderComponent) {
```